docs/sources/explore/index.md

```diff
@@ -49,6 +49,12 @@ In split view, timepickers for both panels can be linked (if you change one, the
 
 You can close the newly created query by clicking on the Close Split button.
 
+## Share shortened link
+
+> Share shortened link is only available in Grafana 7.3 and above.
+
+The Share shortened link feature allows you to create smaller and simpler URLs of the format `/goto/:uid` instead of using and sharing URLs that containin complex query parameters. You can create an copy shortened link by clicking on Share shortened link button in Explore toolbar.
```

Comment:
The Share shortened link capability allows you to create smaller and simpler URLs of the format `/goto/:uid` instead of using longer URLs containing complex query parameters. You can create a shortened link by clicking on the **Share** option in Explore toolbar.

Reply:
Yes, this sounds much better. Updated.